rules/windows/defense_evasion_defender_exclusion_via_powershell.toml

```
@@ -0,0 +1,57 @@
+[metadata]
+creation_date = "2021/07/20"
+maturity = "production"
+updated_date = "2021/07/20"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies modifications to the Windows Defender configuration settings using PowerShell to add exclusions at the folder directory or process level.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Windows Defender Exclusions Added via PowerShell"
+note = """## Triage and analysis
+
+Detections should be investigated to identify if the activity corresponds to legitimate activity used to put in exceptions for Windows Defender. As this rule detects post-exploitation process activity, investigations into this should be prioritized"""
+references = ["https://www.bitdefender.com/files/News/CaseStudies/study/400/Bitdefender-PR-Whitepaper-MosaicLoader-creat5540-en-EN.pdf"]
+risk_score = 21
+rule_id = "2c17e5d7-08b9-43b2-b58a-0270d65ac85b"
+severity = "low"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Defense Evasion"]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+process where event.type == "start" and
+ (process.name : ("powershell.exe", "pwsh.exe") or process.pe.original_file_name == "PowerShell.EXE") and
+ process.parent.name: "cmd.exe" and
+ (process.args : "Add-MpPreference" and process.args : ("-Exclusion*") or
+ process.args : "Set-MpPreference" and process.args : ("-Exclusion*"))
```

these wouldn't work with `powershell -c "..."` because the whole PS command is a single arg (this is what I was referring to on your PR @austinsonger)

you would need to have wildcards, because it's a `keyword` field and not a `text` field.
I think it'd be easier to make a single wildcard or regex. thoughts on this?
```suggestion
  process.args regex~ """.*(Add|Set)\-MpPreference.*\-Exclusion.*"""
```

alternatively
```suggestion
  process.args : ("*Add-MpPreference*-Exclusion*", "*Set-MpPreference*-Exclusion*")
```

---

Cool, thanks for the heads up. The second choice looks more cleaner to me from readability standpoint.